modules/swagger-parser-safe-url-resolver/src/main/java/io/swagger/parser/v3/urlresolver/PermittedUrlsChecker.java

```diff
@@ -0,0 +1,97 @@
+package io.swagger.parser.v3.urlresolver;
```

Comment:
To be consistent with the project/artifacts package naming this should be `io.swagger.v3.parser...`
Applicable to all files

Reply:
Updated across the packages now